addons/portal/static/src/scss/portal.scss

```diff
@@ -362,7 +362,7 @@ img, .media_iframe_video, .o_image {
                 @include o-position-absolute(0, $left: 0);
                 line-height: $line-height-base;
 
-                + span, + div {
+                + span, + div, + a {
                     display: block;
```

Comment:
Apparently we need a `text-truncate` somewhere

![image](https://user-images.githubusercontent.com/8558351/231212006-05c9b5f1-1a55-4db3-9719-b53989d2b1ec.png)